api-audit/src/main/java/com/capitalone/dashboard/evaluator/RegressionTestResultEvaluator.java

```
@@ -291,50 +255,111 @@ private long getEpochChangeDate(Feature feature) {
+     * @param testResult
+     * @return featureTestResultMap
+     */
+    protected HashMap getFeatureTestResult(TestResult testResult) {
+        HashMap<String,Integer> featureTestResultMap = new HashMap<>();
+        featureTestResultMap.put(SUCCESS_COUNT, testResult.getSuccessCount());
+        featureTestResultMap.put(FAILURE_COUNT, testResult.getFailureCount());
+        featureTestResultMap.put(SKIP_COUNT, testResult.getSkippedCount());
+        featureTestResultMap.put(TOTAL_COUNT,testResult.getTotalCount());
+        return featureTestResultMap;
     }
 
-    private Dashboard getDashboardDetails(String title, DashboardType dashboardType) {
-        Dashboard dashboardDetails = dashboardRepository.findByTitleAndType(title, dashboardType);
+    /**
+     * update test result audit statuses
+     * @param testResult
+     * @param testResultsAuditResponse
+     * @return
+     */
+    private TestResultsAuditResponse updateTestResultAuditStatuses(TestResult testResult, TestResultsAuditResponse testResultsAuditResponse) {
+
+        boolean isSuccessHighPriority = settings.getTestResultSuccessPriority().equalsIgnoreCase(PRIORITY_HIGH);
+        boolean isFailureHighPriority = settings.getTestResultFailurePriority().equalsIgnoreCase(PRIORITY_HIGH);
+        boolean isSkippedHighPriority = settings.getTestResultSkippedPriority().equalsIgnoreCase(PRIORITY_HIGH);
+
+        if (isFailureHighPriority){
+            if (testResult.getFailureCount() > NumberUtils.INTEGER_ZERO) {
+                testResultsAuditResponse.addAuditStatus(TestResultAuditStatus.TEST_RESULT_AUDIT_FAIL);
+            } else if (testResult.getFailureCount() == NumberUtils.INTEGER_ZERO && testResult.getSuccessCount() > NumberUtils.INTEGER_ZERO) {
+                testResultsAuditResponse.addAuditStatus(TestResultAuditStatus.TEST_RESULT_AUDIT_OK);
+            } else if (testResult.getFailureCount() == NumberUtils.INTEGER_ZERO && testResult.getSuccessCount() == NumberUtils.INTEGER_ZERO
```

Since this is the same for both cases consider moving it to the top. Fail fast.

---

@Sbrenthughes moved the skipped check to the top, also updated the logic to use 95% threshold due to design change from PO.